README.md

```diff
@@ -82,6 +82,8 @@ class Llama(models.Model):
 
 Now we could setup some endpoints to `create/update` a `Llama` instance for example.
 
+We have included in `fields.py` in the project a field util called `UploadedFilePrimaryKeyRelatedField`. This may be used in the create serializer for your endpoint to ensure only users that have uploaded the file are able to assign this to a custom model.
```

Comment:
As this is part of the `Examples`, I think it makes more sense to explain this from the package users point of view. Something like

```
If needed, you can import a util `UploadedFilePrimaryKeyRelatedField ` from `s3_file_uploads.fields`. This may be used...
```